Assets/MixedRealityToolkit/Services/MixedRealityToolkit.cs

```diff
@@ -539,7 +539,29 @@ private void EnsureMixedRealityRequirements()
         /// <summary>
         /// Returns the Singleton instance of the classes type.
         /// </summary>
-        public static MixedRealityToolkit Instance => activeInstance;
+        public static MixedRealityToolkit Instance
+        {
+            get
+            {
+                if (activeInstance != null)
+                {
+                    return activeInstance;
+                }
+
+                var mrtks = FindObjectsOfType<MixedRealityToolkit>();
+                for (int i = 0; i < mrtks.Length; i++)
+                {
+                    RegisterInstance(mrtks[i]);
+                }
+
+                if (activeInstance == null)
+                {
+                    Debug.LogError("Didn't find / couldn't create an active instance of mixed reality toolkit. You can add one to your scene via 'Mixed Reality Toolkit->Add to Scene and Configure'");
+                }
+
```

Comment:
Do you know if this is something that's editor only (i.e. and non-play mode only)? It seems like there's some weird non-determinism here around which would also get added as the activeInstance based on enumeration order of MixedRealityToolkit instances.

I don't really have enough background here to state if this is safe or not for all unity player (i.e. actual device) cases, and I'd probably defer to @Railboy for this

Reply:
Fixed by using Lars's suggestion to not actually run this check except when in editor.